<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Equipos_Detalles</class>
 <widget class="QWidget" name="Equipos_Detalles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detalle Equipos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="titulo_equipos">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Equipos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_equipos"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Informacion del Equipo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="tree_info_equipo">
           <column>
            <property name="text">
             <string>equipos</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Datos Tratados</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="tree_info_datos">
           <column>
            <property name="text">
             <string>datos</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
